client/src/app/site/agenda/models/view-item.ts

```diff
@@ -26,6 +26,10 @@ export class ViewItem extends BaseViewModel {
         return this.item ? this.item.speakerAmount : null;
     }
 
+    // public get is_internal(): boolean {
```

Comment:
For version 2.3 we added a third item type. Now an item can be public hidden and intern. I would propose calling the column in the csv `Item type` with `public item`, `internal item` and `Hidden item` as values. (See https://github.com/OpenSlides/OpenSlides/blob/387c11cd84900b3c7f9ea93a9bc749dee7229a4f/client/src/app/shared/models/agenda/item.ts#L17)
In this file please add a `get type` which returns the type number and `get verboseType` which returns the name of the type. You can use the `verboseType` during the export.